cli/command/admin_test.go

```diff
@@ -21,3 +22,28 @@ func TestDebugAdmin(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestReadLayer0InstanceLogs(t *testing.T) {
```

Comment:
This should follow our existing naming patterns, e.g. with the instanceLog->log conversion, this would be `TestAdminLogs`